x-pack/plugins/lens/public/datatable_visualization/components/cell_value.tsx

```diff
@@ -75,7 +77,7 @@ export const createGridCell = (
          */
         dangerouslySetInnerHTML={{ __html: content }} // eslint-disable-line react/no-danger
         data-test-subj="lnsTableCellContent"
-        className={`lnsTableCell ${alignmentClassName}`}
+        className={`${tableCellClassName} ${alignmentClassName}`}
```

Comment:
nit: please don't concatenate class names. Normally we use `classNames` for that cases
 
example how to use classNames with `Dynamic` class names:

```
let buttonType = 'primary';
classNames({ [`btn-${buttonType}`]: true });
```

Reply:
done